Vagrantfile

```diff
@@ -78,7 +77,6 @@ Vagrant.configure(2) do |config|
   config.vm.provision "shell", inline: "sudo systemctl daemon-reload", run: "always"
   config.vm.provision "shell", inline: "sudo service apache2 restart", run: "always"
   config.vm.provision "shell", inline: "sudo service mysql restart", run: "always"
-  config.vm.provision "shell", inline: "sudo service uploader restart", run: "always"
```

Comment:
Thank you, I ran into this one when I was testing locally and was going to make a followup PR but you're super on it.